e2e/serial/send/2_shortcuts-sendFlow.test.ts

```diff
@@ -0,0 +1,234 @@
+import 'chromedriver';
+import 'geckodriver';
+import { Key, WebDriver } from 'selenium-webdriver';
+import {
+  afterAll,
+  afterEach,
+  beforeAll,
+  beforeEach,
+  describe,
+  expect,
+  it,
+} from 'vitest';
+
+import {
+  checkExtensionURL,
+  checkWalletName,
+  delayTime,
+  executePerformShortcut,
+  findElementByTestId,
+  findElementByText,
+  getExtensionIdByName,
+  getRootUrl,
+  goToPopup,
+  importWalletFlowUsingKeyboardNavigation,
+  initDriverWithOptions,
+  isElementFoundByText,
+  navigateToElementWithTestId,
+  takeScreenshotOnFailure,
+  transactionStatus,
+} from '../../helpers';
+import { TEST_VARIABLES } from '../../walletVariables';
+
+let rootURL = getRootUrl();
+let driver: WebDriver;
+
+const browser = process.env.BROWSER || 'chrome';
+const os = process.env.OS || 'mac';
+
+describe('Complete send flow via shortcuts and keyboard navigation', () => {
+  beforeAll(async () => {
+    driver = await initDriverWithOptions({
+      browser,
+      os,
+    });
+    const extensionId = await getExtensionIdByName(driver, 'Rainbow');
+    if (!extensionId) throw new Error('Extension not found');
+    rootURL += extensionId;
+  });
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  beforeEach(async (context: any) => {
+    context.driver = driver;
+  });
```

Comment:
`beforeEach`/`afterEach` have a generic for the context type

```suggestion
  beforeEach<{ driver: WebDriver }>((context) => {
    context.driver = driver;
  });
```